Src/Plugins/Security/Security.csproj

```diff
@@ -16,12 +16,19 @@
     <Content Include="build\Sarif.PatternMatcher.Security.targets">
       <PackagePath>build\</PackagePath>
     </Content>
+    <Content Include="Security.SharedStrings.txt">
```

Comment:
>Security.SharedStrings.txt [](start = 22, length = 26)

do we need to change the targets file as well?

Reply:
Yes!

---
In reply to: [553691486](https://github.com/microsoft/sarif-pattern-matcher/pull/90#discussion_r553691486) [](ancestors = 553691486)

Reply:
Fixed it! Please merge if you are satisfied.

---
In reply to: [553726183](https://github.com/microsoft/sarif-pattern-matcher/pull/90#discussion_r553726183) [](ancestors = 553726183,553691486)